src/Node/Stream.purs

```diff
@@ -61,6 +65,18 @@ type Writable r = Stream (write :: Write | r)
 -- | A duplex (readable _and_ writable stream)
 type Duplex = Stream (read :: Read, write :: Write)
 
+data Chunk
+data Data = BufferData Buffer
+          | StringData String
+          | UnknownData Chunk
```

Comment:
I don't mind, but it might make the code a bit harder to read because of the ambiguity of `Foreign`.